torchtext/experimental/models/xlmr_transform.py

```diff
@@ -0,0 +1,29 @@
+import torch.nn as nn
+from typing import List
+from .utils import load_model_from_url
+
+
+class XLMRTransform(nn.Module):
+    """XLM-R encode transform."""
+
+    def __init__(self, tokenizer, vocab):
+        super(XLMRTransform, self).__init__()
+        self.tokenizer = tokenizer
+        self.vocab = vocab
+
+    def forward(self, input_src: str) -> List[int]:
+        return self.vocab(self.tokenizer(input_src))
+
+
+def load_xlmr_transform():
```

Comment:
@datumbox Here is a pretrained transform case for the text domain.

Reply:
Sounds good. This should be supported fine for presets.

The only question that I have is over the use of `load_model_from_url`. Do you typically store the whole model or only its weights? As far as I've seen the latter it typically preferred but maybe that does not work for you if the definition of the model is on an external lib.

Reply:
Yes. For the model itself, we store weights. However, for those two transforms, they are not really `nn.Module` so we have to store the whole models. Happy to hear a better solution.